pymc/data.py

```diff
@@ -562,35 +564,47 @@ def Data(
     """Data container that registers a data variable with the model.
 
     Depending on the ``mutable`` setting (default: True), the variable
-    is registered as a ``SharedVariable``, enabling it to be altered
-    in value and shape, but NOT in dimensionality using ``pm.set_data()``.
+    is registered as a :class:`~aesara.compile.sharedvalue.SharedVariable`,
+    enabling it to be altered in value and shape, but NOT in dimensionality using
+    :func:`pymc.set_data`.
+
+    To set the value of the data container variable, check out
+    :func:`pymc.Model.set_data`.
+
+    For more information, take a look at this example notebook
+    https://docs.pymc.io/projects/examples/en/latest/pymc3_howto/data_container.html
```

Comment:
We should not use html links to refer to pymc (or other python libraries commonly used like arviz, aesara, numpy...). This should be done with intersphinx:

```
:ref:`nb:data_container`
```

Which will show the title of the notebook as link text, so it's probably best to update the text something like "read the <link here> notebook"